src/grading-settings/grading-sidebar/GradingSidebar.test.jsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { IntlProvider, injectIntl } from '@edx/frontend-platform/i18n';
 import { render } from '@testing-library/react';
-
+import { AppProvider } from '@edx/frontend-platform/react';
+import { initializeMockApp } from '@edx/frontend-platform';
+import initializeStore from '../../store';
```

Comment:
Please, add an empty line between library imports and local imports

```suggestion
import { initializeMockApp } from '@edx/frontend-platform';

import initializeStore from '../../store';
```